benchexec/tablegenerator/react-table/src/components/ReactTable.js

```diff
@@ -619,8 +621,8 @@ const Table = (props) => {
     }
   }, [columnResizing, columnsResizeValues]);
 
-  // get selected status and category values
-  useEffect(() => {
+  // get new Filtered Column Values
+  function getNewFileteredColumnValues() {
```

Comment:
Can you please rename to `getNewFilteredColumnValues()`? (typo)

Also, the comment doesn't really describe something, it is just the same as the function name, so it is redundant. Can you make it more descriptive?